org.eclipse.jdt.debug.ui/ui/org/eclipse/jdt/internal/debug/ui/JavaHotCodeReplaceListener.java

```diff
@@ -131,15 +135,15 @@ public void obsoleteMethods(final IJavaDebugTarget target) {
 		final String message= NLS.bind(DebugUIMessages.JDIDebugUIPlugin__0__contains_obsolete_methods_1, new Object[] {vmName});
 		final IStatus status= new Status(IStatus.WARNING, JDIDebugUIPlugin.getUniqueIdentifier(), IStatus.WARNING, DebugUIMessages.JDIDebugUIPlugin_Stepping_may_be_hazardous_1, null);
 		final String toggleMessage= DebugUIMessages.JDIDebugUIPlugin_2;
+		final String toggleMessage2 = DebugUIMessages.JDIDebugUIPlugin_5;
```

Comment:
nitpick: the whitespaces before the assignment operator are not uniform.

Reply:
Will make it uniform,